lib/mix/tasks/nerves_hub.org.ex

```diff
@@ -0,0 +1,191 @@
+defmodule Mix.Tasks.NervesHub.Org do
+  use Mix.Task
+
+  import Mix.NervesHubCLI.Utils
+
+  alias Mix.NervesHubCLI.Shell
+
+  @shortdoc "Manages an organization"
+
+  @moduledoc """
+  Manages an organization
+
+  # Managing user roles
+
+  The following functions allow the management of user roles within your organization.
+  Roles are a way of granting users a permission level so they may perform
+  actions for your org. The following is a list of valid roles in order of
+  highest role to lowest role:
+
+    :admin
+    :delete
+    :write
+    :read
+
+  NervesHub will validate all actions with your user role. If an action you are
+  trying to perform requires :write, the user performing the action will be
+  required to have an org role of :write or higher (:admin, :delete).
+
+  Managing user roles in your org will require that your user has the org role of
+  :admin.
+
+  ## user list
+
+  List the users and their role for the organization.
+
+    mix nerves_hub.org user list
```

Comment:
This needs 2 more spaces to render as code. I could swear that `ex_doc` changed at some point on this, but try looking at the docs and you see how stuff like this gets messed up now.